packages/apollo/lib/drivers/apollo-base.driver.ts

```diff
@@ -67,11 +73,7 @@ export abstract class ApolloBaseDriver<
         typeof options.playground === 'object' ? options.playground : undefined;
       defaults = {
         ...defaults,
-        plugins: [
-          ApolloServerPluginLandingPageGraphQLPlayground(
-            playgroundOptions,
-          ) as PluginDefinition,
-        ],
+        plugins: [ApolloServerPluginLandingPageLocalDefault(playgroundOptions)],
```

Comment:
AFAIK we should use `@apollo/server-plugin-landing-page-graphql-playground` here instead